File: onebot/src/main/java/cn/evole/onebot/sdk/event/request/GroupAddRequestEvent.java
package cn.evole.onebot.sdk.event.request;

import com.google.gson.annotations.SerializedName;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;
import lombok.experimental.SuperBuilder;

/**
 * Created on 2022/7/8.
 *
 * @author cnlimiter
 */
@Data
@NoArgsConstructor
@AllArgsConstructor
@SuperBuilder(toBuilder = true)
@EqualsAndHashCode(callSuper = true)
public class GroupAddRequestEvent extends RequestEvent {

    @SerializedName( "sub_type")
    public String subType;

    @SerializedName( "group_id")
    public long groupId;

}

File: onebot/src/main/kotlin/sdk/event/request/GroupAddRequestEvent.kt
package cn.evolvefield.onebot.sdk.event.request

import com.google.gson.annotations.SerializedName

class GroupAddRequestEvent : RequestEvent() {
    @SerializedName("sub_type")
    var subType = ""
    @SerializedName("group_id")
    var groupId = 0L
    @SerializedName("invitor_id")
    var invitorId = 0L
}
